### DISCUSSION TITLE
MySQL Version - Herd Pro

### DISCUSSION THREAD

**Discussion by discoveryjames** [beyondcode/herd-community#466] [Q&A]
It's nice to be able to match the MySQL version used in production, but currently it only offers one version.

Is there a way or are there plans to allow for other versions?

**Comment by mpociot** [comment]
We wanted to stick to the versions that Laravel Forge offers. Which one is missing for you?

**Comment by discoveryjames** [comment]
Thanks for the reply!

We use a range, but we're only seeing 1 at the moment, which is 8.0.36 - should there be more than one?

**Reply by cyppe** [reply]
I don't think it's worth having a lot of minor versions. 
But it would be nice to have access to latest major version, for example 8.3.x that is out now and of course also continue with 8.0.x as it's still most common in the wild.

**Comment by adiachenko** [comment]
@mpociot MySQL 8.4 is the current LTS. If you don’t plan to offer a selection of major MySQL versions, would you at least consider periodically updating the single version you support?

MySQL 8.1 was released in 2023, yet Herd still only provides 8.0.36 according to the [documentation](https://herd.laravel.com/docs/macos/herd-pro-services/mysql#versions), which, in my opinion, hurts the appeal of the paid version. It’s also worth noting thatsome major services like [MySQL HeatWave](https://docs.oracle.com/en/database/mysql/heatwave-aws/mysql-version-support-schedule.html) are even scheduled to drop support for 8.0 this month.

**Reply by mpociot** [reply]
MySQL 8.4 should also be available when creating a new database service 🤔

**Reply by adiachenko** [reply]
For clarity, I haven’t tried Herd Pro yet, so this might just be a [documentation issue](https://herd.laravel.com/docs/macos/herd-pro-services/mysql#versions). If that’s the case, I’d kindly suggest updating the documentation to help avoid any confusion.
